exasol/nb_connector/text_ai_extension_wrapper.py

```diff
@@ -173,76 +199,119 @@ def initialize_text_ai_extension(
+    conf.save(CKey.txaie_models_bfs_dir, BFS_MODELS_DIR)
+    conf.save(CKey.txaie_models_cache_dir, MODELS_CACHE_DIR)
+
+    if install_slc:
+        print("Text AI: Downloading and installing Script Language Container (SLC)")
+        if container_file:
+            deploy_slc(container_file=container_file)
         else:
-            deploy_language_container(
-                conf=conf,
-                path_in_bucket=PATH_IN_BUCKET,
-                language_alias=language_alias,
-                activation_key=ACTIVATION_KEY,
-                container_file=container_file,
-                container_name=container_name,
-                allow_override=allow_override,
-            )
+            version = version or importlib.metadata.version("exasol_text_ai_extension")
+            deploy_slc(version=version)
 
-    if run_upload_models:
+    if install_models:
         #  Install default Hugging Face models into the Bucketfs using
         #  Transformers Extensions upload model functionality.
-        raise NotImplementedError("Implementation is waiting for TE release.")
+        print("Text AI: Downloading and installing Huggingface models to BucketFS:")
+        install_model(
+            conf,
+            TransformerModel(
+                DEFAULT_FEATURE_EXTRACTION_MODEL, "feature-extraction", AutoModel
+            ),
+        )
+        install_model(
+            conf,
+            TransformerModel(
+                DEFAULT_NAMED_ENTITY_MODEL,
+                "token-classification",
+                AutoModelForTokenClassification,
+            ),
+        )
+        install_model(
+            conf,
+            TransformerModel(
+                DEFAULT_NLI_MODEL,
+                "zero-shot-classification",
+                AutoModelForSequenceClassification,
+            ),
+        )
 
-    if run_deploy_scripts:
-        raise NotImplementedError(
-            "Currently there are no Text-AI specific scripts to deploy."
+    if install_udf_scripts:
+        print("Text AI: Creating UDF scripts")
+        pyexasol_connection = open_pyexasol_connection(
+            conf, schema=conf.get(CKey.db_schema)
         )
+        create_scripts(pyexasol_connection)
 
-    if run_encapsulate_bfs_credentials:
+    if install_bfs_credentials:
+        print(f"Text AI: Creating BFS connection {bfs_conn_name}")
         encapsulate_bucketfs_credentials(
             conf, path_in_bucket=PATH_IN_BUCKET, connection_name=bfs_conn_name
         )
+    print(f"Text AI: Installation finished.")
+
+
+class Extraction(AbstractExtraction):
+    def defaults_with_model_repository(self, conf: Secrets) -> Defaults:
+        if self.defaults.model_repository:
+            return self.defaults
+        model_repository = BucketFSRepository(
+            connection_name=conf.txaie_bfs_connection,
+            sub_dir=conf.txaie_models_bfs_dir,
+            # connection_name=conf.te_bfs_connection,
+            # sub_dir=conf.te_models_bfs_dir,
```

Comment:
here we need to use, whatever we are using for install_model, and I think, for the moment it is txaie_...

```suggestion
```